logo.svg

```diff
@@ -26,7 +26,17 @@
                 <path d="M169.350586,22.1777344 L173.62793,22.1777344 L173.62793,13.3886719 C173.62793,11.1035156 174.858398,9.84375 176.967773,9.84375 C177.583008,9.84375 178.168945,9.93164062 178.520508,10.078125 L178.520508,6.31347656 C178.198242,6.22558594 177.758789,6.15234375 177.290039,6.15234375 C175.458984,6.15234375 174.18457,7.28027344 173.71582,9.21386719 L173.62793,9.21386719 L173.62793,6.47460938 L169.350586,6.47460938 L169.350586,22.1777344 Z"></path>
                 <path d="M182.724609,27.3486328 C186.210938,27.3486328 188.320312,26.1621094 189.550781,22.4560547 L194.941406,6.47460938 L190.429688,6.47460938 L187.104492,18.75 L186.987305,18.75 L183.676758,6.47460938 L178.974609,6.47460938 L184.526367,22.1777344 L184.335938,22.6904297 C183.955078,23.7597656 183.251953,24.1259766 181.977539,24.1259766 C181.347656,24.1259766 180.849609,24.0380859 180.541992,23.9794922 L180.541992,27.1582031 C181.040039,27.2460938 181.889648,27.3486328 182.724609,27.3486328 Z"></path>
             </g>
-            <path d="M32.6137819,33.8444602 L2.54969389,33.8444602 C1.14153684,33.8444602 0,32.6769536 0,31.2367612 C0,30.7913947 0.11152861,30.3534497 0.32395522,29.9646699 L15.3559992,2.45326441 C16.0429283,1.19605782 17.5962915,0.7464252 18.825533,1.44898181 C19.237395,1.68437608 19.5773184,2.03203256 19.8074766,2.45326441 L34.8395205,29.9646699 C35.5264496,31.2218764 35.0868185,32.8105785 33.857577,33.5131351 C33.4774452,33.7303944 33.0492417,33.8444602 32.6137819,33.8444602 Z M28.1160234,17.094738 C35.2111906,13.0647288 44.1049062,14.3293322 49.8178648,20.1805375 C55.5308233,26.0317428 56.6451013,35.0173145 52.5368502,42.1065197 C48.4285992,49.195725 40.1174634,52.6290638 32.2565807,50.4843201 C24.3956981,48.3395766 18.9360944,41.1490635 18.9344262,32.9385181" stroke="#000000" stroke-width="4"></path>
+            <path class="figures" d="M32.6137819,33.8444602 L2.54969389,33.8444602 C1.14153684,33.8444602 0,32.6769536 0,31.2367612 C0,30.7913947 0.11152861,30.3534497 0.32395522,29.9646699 L15.3559992,2.45326441 C16.0429283,1.19605782 17.5962915,0.7464252 18.825533,1.44898181 C19.237395,1.68437608 19.5773184,2.03203256 19.8074766,2.45326441 L34.8395205,29.9646699 C35.5264496,31.2218764 35.0868185,32.8105785 33.857577,33.5131351 C33.4774452,33.7303944 33.0492417,33.8444602 32.6137819,33.8444602 Z M28.1160234,17.094738 C35.2111906,13.0647288 44.1049062,14.3293322 49.8178648,20.1805375 C55.5308233,26.0317428 56.6451013,35.0173145 52.5368502,42.1065197 C48.4285992,49.195725 40.1174634,52.6290638 32.2565807,50.4843201 C24.3956981,48.3395766 18.9360944,41.1490635 18.9344262,32.9385181" stroke="#000000" stroke-width="4"></path>
         </g>
+        <style>
+            @media (prefers-color-scheme: dark) {
+                .figures {
+                    stroke: #c9d1d9;
+                }
+                .text {
+                    fill: #c9d1d9;
```

Comment:
```suggestion
                    stroke: #ffffff;
                }
                .text {
                    fill: #ffffff;
```

Reply:
Good evening!
Thank you very much for your prompt reply. This is my first open source PR. Tell me, do I need to squash my commits after each reviewer's comments every time?

Reply:
> Tell me, do I need to squash my commits after each reviewer's comments every time?

Nope, all good, we commonly squash PRs anyway.